source/Launchpad-Configuration/ApplicationConfiguration.class.st

```diff
@@ -192,6 +192,22 @@ ApplicationConfiguration >> synchronizeValuesWith: resolvedValues [
 	values := resolvedValues
 ]
 
+{ #category : #'reflective operations' }
+ApplicationConfiguration >> valueAt: aKey [
+	"For configuration access without DNU handling, provide standard Dictionary api. 
+	With normal configuration parameters, there should never be an absent value, but answer nil just in case"
+	
+	^self valueAt: aKey ifAbsent: [ nil ]
```

Comment:
as this is never expected to have an absent value (if you are using the framework properly) then exception a good suggestion as its indicative of something very bad

Reply:
would you use "self missingConfigureationFor: " or just a normal dictionary missing error from #at: ?

Reply:
I think it's best to let the default key not found value exception because AFAIR missingConfigurationFor: raises an exception to notify the case that a valid mandatory parameter is missing.